travis/export_env_spark_3.0_hadoop_3.2.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+echo "export HADOOP_HOME=/opt/hadoop-3.2.1" >> ~/.branchrc
```

Comment:
I believe we don't need to add this environment in branchrc, because for every job/stage, we will export it.

Just `export HADOOP_HOME/SPARK_HOME` will be fine.

Reply:
@gczsjdy  Hadoop startup is started by ssh command , so  it will create new shell .

Reply:
Great!

Reply:
Let's recheck this, `branchrc`?